content/_data/changelogs/weekly.yml

```diff
@@ -7719,6 +7719,134 @@
 
   # pull: 4732 (PR title: Fix hyperlink in maintainers doc)
 
+- version: "2.239"
+  date: 2020-06-01
+  banner: >
+    Jenkins 2.239 was not packaged or delivered.
+    All changes planned for 2.239 are included in 2.240.
+  changes:
+  - type: bug
+    category: internal
+    authors:
+    - kohsuke
+    message: |-
+      Jenkins 2.239 was not placed in the artifact repository or on the download site.
+
+- version: "2.240"
+  date: 2020-06-01
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4752
+    issue: 60924
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+    message: |-
+      Release 'alpha' dark theme.
+  - type: major rfe
+    category: developer
+    pull: 4752
+    issue: 60924
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+    message: |-
+      Developer: Add incubating support for CSS variables, useful in themes.
```

Comment:
I do not think it is incubating. Dark Theme is incubating, variables are there to stay

Reply:
Removed "incubating" in 9d6ac5d829974359678f86e963b530b07c76acb9